omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_10/OrderResource1_10.java

```diff
@@ -102,6 +108,43 @@ public DelegatingResourceDescription getRepresentationDescription(Representation
 		}
 	}
 	
+	@Override
+	public Model getCREATEModel(Representation rep) {
```

Comment:
I don't think they commented on this. If you prefer `getCreateModel` over `getCREATEModel`, I can refactor.